released/packages/coq-topology/coq-topology.10.1.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "palmskog@gmail.com"
+
+homepage: "https://github.com/coq-community/topology"
+dev-repo: "git+https://github.com/coq-community/topology.git"
+bug-reports: "https://github.com/coq-community/topology/issues"
+license: "LGPL-2.1-or-later"
+
+synopsis: "General topology in Coq"
+description: """
+This library develops some of the basic concepts
+and results of general topology in Coq.
+"""
+
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune" {>= "2.5"}
+  "coq" {(>= "8.12" & < "8.18~") | (= "dev")}
```

Comment:
```suggestion
  "coq" {(>= "8.12" & < "8.19~") | (= "dev")}
```
It turns out that this is compatible with the soon-to-be-released Coq 8.18, so can we relax this in the interest of less work later?

Reply:
So it won't be problematic, that the opam files in the coq-topology repo at tag `v10.1.0` contain the old restriction `< "8.18~"`?

Reply:
People (regular Coq users, at least) don't use opam files inside repos when they install libraries - they use the Coq opam `released` repo. So I think we view opam files inside tags as overridden by opam files here, i.e. the files here are canonical.